release/publish_release.sh

```diff
@@ -227,7 +227,7 @@ fi
 if [[ "${DO_GCS}" == "true" ]]; then
   echo "Copying to GCS destination ${GCS_DEST}"
   gsutil -m cp "${UPLOAD_DIR}"/deb/istio*.deb "gs://${GCS_DEST}/deb/"
-  gsutil -m cp "${UPLOAD_DIR}/SHA256SUMS"     "gs://${GCS_DEST}/"
+# gsutil -m cp "${UPLOAD_DIR}/SHA256SUMS"     "gs://${GCS_DEST}/"
```

Comment:
why are we removing this ?

Reply:
unfinished, will finish and add it in a subsequent PR